tools/marvin/marvin/config/test_data.py

```diff
@@ -1617,19 +1617,28 @@
                 "password": "password",
             },
             "ldap_account": {
-                "email": "",
-                "firstname": "",
-                "lastname": "",
-                "username": "",
-                "password": "",
+                "email": "hello@world.com",
+                "firstname": "hello",
+                "lastname": "world",
+                "username": "helloworld",
+                "password": "helloworld",
             },
             "ldap_configuration": {
-                "basedn": "",
-                "emailAttribute": "",
-                "userObject": "",
-                "usernameAttribute": "",
-                "hostname": "",
-                "port": "",
+                "basedn": "CN=Users,DC=hyd-qa,DC=com",
+                "emailAttribute": "mail",
+                "userObject": "user",
+                "usernameAttribute": "sAMAccountName",
+                "hostname": "10.147.38.163",
```

Comment:
Hardcoding these values will prevent anyone else than Citrix from running this test. Let's make sure anyone can run this test instead, or else it will not have the desired effect. Having a few requirements is fine, but at least make it generic.